xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -322,9 +339,18 @@ The JSON token value represents a number less than <see cref="F:System.Byte.MinV
       <Docs>
         <summary>Parses the current JSON token value from the source and decodes the Base64 encoded JSON string as a byte array.</summary>
         <returns>The byte array that represents the current JSON token value.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The `GetBytesFromBase64` method throws an exception if the entire UTF-8 encoded token value cannnot be decoded to a <xref:System.Byte[]> value.
```

Comment:
```suggestion
The `GetBytesFromBase64` method throws an exception if the entire UTF-8 encoded token value cannot be decoded to a <xref:System.Byte[]> value.
```

Reply:
We should see how System.Byte[] renders after the build finishes.

Reply:
Doesn't render well:
![image](https://user-images.githubusercontent.com/6527137/64576197-559ec100-d32c-11e9-8b5f-88919a5919c3.png)

Any suggestions?

Reply:
Then maybe we should simply use
```suggestion
The `GetBytesFromBase64` method throws an exception if the entire UTF-8 encoded token value cannnot be decoded to a byte array.
```

Reply:
@ahsonkhan I added this suggestion after approving. You might want to commit this too.

Reply:
Looks like I missed this.